packages/core/entity/Annotation/index.ts

```diff
@@ -81,6 +82,10 @@ export default class Annotation {
         return this.annotation.isOpenFileLink || false;
     }
 
+    public get isImmutable(): boolean {
```

Comment:
wondering if would benefit from an explanatory comment here (e.g., that this is mainly to prevent editing on certain top level annotations)